linkerd/app/inbound/src/http/mod.rs

```diff
@@ -295,29 +304,29 @@ pub mod fuzz_logic {
         let server = build_fuzz_server(cfg, rt, profiles, connect).new_service(accept);
         let (mut client, bg) = http_util::connect_and_accept(&mut client, server).await;
 
-        let http_method = if is_get {
-            http::Method::GET
-        } else {
-            http::Method::POST
-        };
-
-        if let Ok(req) = Request::builder()
-            .method(http_method)
-            .uri(uri)
-            .header(header_name, header_value)
-            .body(Body::default())
-        {
-            let rsp = http_util::http_request(&mut client, req).await;
-            let _body = http_util::body_to_string(rsp.into_body()).await;
+        // Now send all of the packets
+        for inp in packets.iter() {
```

Comment:
this is nitpicky, but I would probably not refer to the HTTP requests as "packets" --- i would expect a value named `packets` to be a vector of individual IP-layer packets, rather than HTTP requests.

maybe call the variable `requests`, instead?

Reply:
Agreed!